pkgs/development/libraries/atkmm/default.nix

```diff
@@ -1,19 +1,19 @@
-{ lib, stdenv, fetchurl, atk, glibmm, pkg-config, gnome3 }:
+{ lib, stdenv, fetchurl, atk, glibmm, pkg-config, meson, ninja, python3, gnome3 }:
 
 stdenv.mkDerivation rec {
   pname = "atkmm";
-  version = "2.36.0";
+  version = "2.28.1";
```

Comment:
Is this an intentional downgrade?

Reply:
Ah, I see now.  I was looking just in commit messages first.

Reply:
Yeah, 2.36.0 introduces a new ABI, afaict we don't need to introduce it at all yet for 40.